sidebars.js

```diff
@@ -106,16 +106,24 @@ const sidebars = {
 
   ],
   subnets: [
-    {
-      type: 'autogenerated',
-      dirName: 'subnets',
-    },
+    'subnets/README',
+    'subnets/create-a-local-subnet',
+    'subnets/create-avm-blockchain',
```

Comment:
Patrick said to remove this from the sidebar

Reply:
Done: https://github.com/ava-labs/avalanche-docs/pull/741/commits/2cfcc8d431592e599106401b4212ec767adf894f